pygenomeworks/bin/evaluate_paf

```diff
@@ -186,58 +187,82 @@ def evaluate_paf(truth_paf_filepath, test_paf_filepath, pos_tolerance, min_recip
 
         # seen_test_overlap_keys.add(key)
         # seen_test_overlap_keys.add(key_reversed)
-
+        #incorrect_query_start += not match_statistics["query_start_valid"]
```

Comment:
Minor comment - can the commented code lines be removed?

Reply:
Oops, yep!